index.html

```diff
@@ -4,11 +4,168 @@
     <title>Mockup Home</title>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
+    <link rel="stylesheet" href="/styles/styles.css"/>
+    <link href="https://fonts.googleapis.com/css?family=Open+Sans&display=swap" rel="stylesheet">
+    <script src="https://kit.fontawesome.com/a076d05399.js"></script>
+    <script src="https://use.fontawesome.com/ca6c48c124.js"></script>
   </head>
   <body>
-    <section id="header"></section>
-    <section id="features"></section>
-    <section id="main"></section>
-    <section id="footer"></section>
+
+    <!-- HEADER SECTION -->
+    <section id="header">
+      <h1 id="page_heading"><a href="index.html">Home Page</a></h1>
+      <h3 id="heading_subtext">The page for the home.</h3>
+      <div id="divider_container">
+        <div id="header_divider" class="divider"></div>
+      </div>
+      <h3 id="features_caption">Take a look at this amazing <strong>home page</strong>!</h3>

```

Comment:
It seems like this font doesn't support a strong or bold style to it. One thing you could do to work around that is to add a black text shadow onto it and space the letters slightly more apart

If you wanna chat about this one I can show you more about it